xbmc/interfaces/json-rpc/schema/methods.json

```diff
@@ -1220,7 +1220,9 @@
       { "name": "disctitle", "$ref": "Optional.String" },
       { "name": "releasedate", "$ref": "Optional.String" },
       { "name": "originaldate", "$ref": "Optional.String" },
-      { "name": "bpm", "$ref": "Optional.Integer" }
+      { "name": "bpm", "$ref": "Optional.Integer" },
+      { "name": "songvideourl", "$ref": "Optional.String" },
+      { "name": "songvideothumb", "$ref": "Optional.String" }
```

Comment:
Just thinking should a songs videothumb is accessed via JSON like all other song art, no special parameter or field for it.